types/mercadopago/models/payment/capturePartial-payload.model.d.ts

```diff
@@ -0,0 +1,7 @@
+export interface CapturePartialPayload {
```

Comment:
Change the filename to kebab-case so it would be: capture-partial-payload.model.d.ts
Cange the interface name to `CapturePartialPaymentPayload` because all the other files follow this pattern